client/app/components/Toast/index.jsx

```diff
@@ -0,0 +1,73 @@
+// @flow
+import React, { useState } from 'react';
+import type { Node } from 'react';
+import { I18n } from 'libs/i18n';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faTimes } from '@fortawesome/free-solid-svg-icons';
+import css from './Toast.scss';
+
+type Props = {
+  alert?: string,
+  notice?: string,
+  appendDashboardClass?: boolean
+};
+export type State = {
+  showToast: boolean,
+};
+
+export const Toast = ({
+  alert, notice, appendDashboardClass,
+}: Props): Node => {
+  const [showAlert, setShowAlert] = useState<boolean>(alert !== null && alert !== '' && !document.documentElement?.hasAttribute('data-turbolinks-preview'));
+  const [showNotice, setShowNotice] = useState<boolean>(notice !== null && notice !== '' && !document.documentElement?.hasAttribute('data-turbolinks-preview'));
+  const hideNotice = () => {
+    setShowNotice(false);
+  };
+  const hideAlert = () => {
+    setShowAlert(false);
+  };
+  if (showAlert || showNotice) {
+    setTimeout(() => {
+      hideNotice();
+      hideAlert();
+    }, 7000);
+  }
+  return (
+    <>
```

Comment:
This fragment is not necessary since there is only one child element.

Reply:
Do you mean that we should convert <> fragment to a div?